pkg/manifests/ext_proc.yaml

```diff
@@ -1,69 +1,154 @@
+    BQAwLTEVMBMGA1UECgwMZXhhbXBsZSBJbmMuMRQwEgYDVQQDDAtleGFtcGxlLmNv
+    bTAeFw0yNDA1MzAxNzM4NDBaFw0zNDA1MjgxNzM4NDBaMC0xFTATBgNVBAoMDGV4
+    YW1wbGUgSW5jLjEUMBIGA1UEAwwLZXhhbXBsZS5jb20wggEiMA0GCSqGSIb3DQEB
+    AQUAA4IBDwAwggEKAoIBAQDJEyqHtNXUVlDM3OtzMBUWcB4OdGC6ZA7JrT24PVVs
+    yUXGCvu3P47HLb6GuW7w7IILBhACdTFNBPQ8pNT2ouItASmaX6oZGVCnud6st/Y+
+    KnV0G406IRA/hZOdBO7hSH4pt//j7iDWPF5OEEwH1LJMXEX/5FdIoDKQHdfJdqHz
+    ANE5LP8RAR/A0hdalvBrDhfMzcVRC7wSGyg1AbXDbo+q7M8xPhGa+95KwcMzj7WX
+    vOcnTcjrKHYTuiJEaINSMo9EEfTEMOp0bgqDtSgdCAQWLUL+p5b59tvYOozbOG5P
+    1CLPzZs8K56AbESBA3tK1dO1fMuvV/oMTVU+IstuflC1AgMBAAGjUzBRMB0GA1Ud
+    DgQWBBRsFJGAAkXFPf6klnBOcZVsTPzBDzAfBgNVHSMEGDAWgBRsFJGAAkXFPf6k
+    lnBOcZVsTPzBDzAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQAM
+    LrFgkTZPXt3zl9WSI22Tk5BT5jgxB+PwDoIsbmd7koXTpAtdFCoSHJQC1c3GQ66i
+    wcTa3ewrqScI62+cMNtgHBYo+j2jMKuND+N2YotzEBlSgsYKua+ehb2n5H7CTvHZ
+    ED9Ch0cP0c4ke4YPW0Xz2hN4SKPwYNVyqaapaW3iQ7zyOPJSPegDbhDRh/soFF5v
+    kDVQC8/fz6VAmPaq+hiem7w8H69FPdPHI3nseqUT+kyUEZkD5eH08MVuQ4uVyYNy
+    cwN8WlDLDCPxxPt9bclj70Xo1/Nae/VSg+rKgfsjwsKweE5gZ7UhWngsjkMVpFzO
+    QPLCCvayjtnIbYbtsXLE
+    -----END CERTIFICATE-----
+---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: instance-gateway-ext-proc
-  namespace: default
+  name: grpc-ext-proc
   labels:
-    app: instance-gateway-ext-proc
+    app: grpc-ext-proc
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: instance-gateway-ext-proc
+      app: grpc-ext-proc
   template:
     metadata:
       labels:
-        app: instance-gateway-ext-proc
+        app: grpc-ext-proc
     spec:
       containers:
-      - name: instance-gateway-ext-proc
-        image: <BUILT-IMAGE>
+      - name: grpc-ext-proc
+        image: us-docker.pkg.dev/kaushikmitra-gke-dev/kaushikmitra-docker-repo/llm-ig-pp-ext-proc 
         args:
-        #TODO: specify label selector and dynamically update pods
-        - -pods
-        - "vllm-78665f78c4-h4kx4,vllm-78665f78c4-hnz84"
         - -podIPs
-        - "10.24.11.6:8000,10.24.5.7:8000"
-        - -enable-fairness
-        - "false"
+        - "10.60.0.7:8000,10.60.1.34:8000,10.60.2.89:8000"
+        - -certPath
+        - "/app/certs"
+        - -v
+        - "3"
         ports:
-        - containerPort: 9002
-      - name: curl
-        image: curlimages/curl
-        command: ["sleep", "3600"]
+        - containerPort: 8000
+        volumeMounts:
+            - name: grpc-ext-proc-secret
+              mountPath: /app/certs
+        readinessProbe:
+              httpGet:
+                path: /healthz
+                port: 8080
+      volumes:
+        - name: grpc-ext-proc-secret
+          secret:
+            secretName: grpc-ext-proc-secret
+            items:
+              - key: tls.crt
+                path: server.crt
+              - key: tls.key
+                path: server.key
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: instance-gateway-ext-proc
+  name: grpc-ext-proc
```

Comment:
I prefer the old name. grpc doesn't describe the purpose - all ext procs are grpc anyway.